python/cudf/cudf/utils/dtypes.py

```diff
@@ -291,14 +291,21 @@ def is_decimal_dtype(obj):
 
 
 def _decimal_normalize_types(*args):
-    s = max([a.dtype.scale for a in args])
-    lhs = max([a.dtype.precision - a.dtype.scale for a in args])
-    p = min(cudf.Decimal64Dtype.MAX_PRECISION, s + lhs)
-    dtype = cudf.Decimal64Dtype(p, s)
-
+    dtype = _find_common_type_decimal([a.dtype for a in args])
     return [a.astype(dtype) for a in args]
 
 
+def _find_common_type_decimal(dtypes):
```

Comment:
Can we call this in `find_common_type` [here](https://github.com/rapidsai/cudf/blob/6136fbc6d0657315a073cf175de0903d6d1bf272/python/cudf/cudf/utils/dtypes.py#L700)? And then avoid importing this underscore function in other modules?